app/helpers/application_helper.rb

```diff
@@ -283,4 +283,12 @@ def first_course_independent?
                 .natural_sort_by(&:title)
                &.first&.term_independent
   end
+
+  def get_announcements
+    return Announcement
+      .where(on_main_page: true, lecture: nil)
+      .pluck(:details)
+      .join
+  end
+
```

Comment:
Thanks, pretty nice concept, these scopes. I've just implemented your suggestion. However, I left the `return` statement in there as it does not do any harm, right? So, it's just more explicit, adds semantic meaning to the code, which is therefore more readable. See also [this discussion](https://stackoverflow.com/questions/1023146/is-it-good-style-to-explicitly-return-in-ruby).

Reply:
No, it does not do any harm. Personally, I do not care whether it is used or not. Only my old buddy, the rubocop, won't like it :-) In the codebase, you will not see many `return`s when they can be omitted.

Reply:
Ok, yeah I still have to install rubocop to format the code ;) Will do this in a next step alongside adjusting the pipeline, so you won't be able to push any code that was incorrectly formatted anymore (otherwise it's really tedious to have different codestyles and also different indentations, e.g. spaces vs. tabs). But still then, rubocop may like a `return` statement if we tell him to do so in the config 😅